book/src/tutorial-3.md

```diff
@@ -52,6 +52,6 @@ fn main() {
 Now, when we run `cargo build`, our bindings to `bzip2` are generated on the
 fly!
 
-[There's more info about `build.rs` files in the crates.io documentation.][build-rs]
+[There's more info about `build.rs` files in Cargo documentation.][build-rs]
```

Comment:
I'd change "in Cargo documentation" for "in the Cargo documentation"

Reply:
I done it @pvdrz